demos/README.md

```diff
@@ -25,6 +25,7 @@ The Open Model Zoo includes the following demos:
 - [Smart Classroom C++ Demo](./smart_classroom_demo/README.md) - Face recognition and action detection demo for classroom environment.
 - [Super Resolution C++ Demo](./super_resolution_demo/README.md) - Super Resolution demo (the demo supports only images as inputs). It enhances the resolution of the input image.
 - [Text Detection C++ Demo](./text_detection_demo/README.md) - Text Detection demo. It detects and recognizes multi-oriented scene text on an input image and puts a bounding box around detected area.
+- [Colorization Python* Demo](./python_demos/colorization/README.md) - Colorization demo. It colorized input frame of video.
```

Comment:
Please, keep the list in alphabetical order.

Reply:
@Wovchena done